tests/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -442,6 +442,49 @@ def test_get_privacy_requests_by_id(
         resp = response.json()
         assert resp == expected_resp
 
+    def test_get_privacy_requests_by_partial_id(
+        self,
+        api_client: TestClient,
+        url,
+        generate_auth_header,
+        privacy_request,
+        postgres_execution_log,
+        mongo_execution_log,
+    ):
+        auth_header = generate_auth_header(scopes=[PRIVACY_REQUEST_READ])
+        response = api_client.get(
+            url + f"?id={privacy_request.id[:5]}", headers=auth_header
```

Comment:
This is mostly copypasta from the test above, testing that with only the first five characters passed into the search, the original privacy request is still returned.